templates/src/error.rs

```diff
@@ -19,6 +19,18 @@ pub enum LoaderError {
         #[snafu(source(from(Vec<fluent_syntax::parser::ParserError>, FluentError::from)))]
         source: FluentError,
     },
+    /// An error was found whilst loading a bundle at runtime.
+    #[snafu(display("Failed to add FTL resources to the bundle"))]
+    FluentBundle {
+        /// The original bundle errors
+        errors: Vec<fluent_bundle::FluentError>
+    },
+}
+
+impl From<Vec<fluent_bundle::FluentError>> for LoaderError {
+    fn from(errors: Vec<fluent_bundle::FluentError>) -> Self {
+        LoaderError::FluentBundle{errors}
+    }
```

Comment:
Please use `snafu::ResultExt` to convert the error instead. https://docs.rs/snafu/0.6.8/snafu/trait.ResultExt.html
```suggestion
```